src/trace/context/extractors/event-bridge.spec.ts

```diff
@@ -106,5 +107,66 @@ describe("EventBridgeEventTraceExtractor", () => {
       const traceContext = extractor.extract(payload);
       expect(traceContext).toBeNull();
     });
+
+    it("extracts trace context from Step Function EventBridge event", () => {
+      // Reset StepFunctionContextService instance
+      StepFunctionContextService["_instance"] = undefined as any;
```

Comment:
Shouldn't this be done before every step?

Reply:
This reset is only required for tests that use the Step Function Context Service. It could be added in a BeforeEach for all tests, but I don't think it would add value.